boards/nxp/mimxrt1160_evk/board.cmake

```diff
@@ -5,13 +5,12 @@
 #
 
 if(CONFIG_SOC_MIMXRT1166_CM7 OR CONFIG_SECOND_CORE_MCUX)
-board_runner_args(pyocd "--target=mimxrt1160_cm7")
 board_runner_args(jlink "--device=MIMXRT1166xxx6_M7" "--reset-after-load")
+board_runner_args(linkserver  "--device=MIMXRT1166xxxxx:MIMXRT1160-EVK")
 elseif(CONFIG_SOC_MIMXRT1166_CM4)
-board_runner_args(pyocd "--target=mimxrt1160_cm4")
 # Note: Please use JLINK above V7.50 (Only support run cm4 image when debugging due to default boot core on board is cm7 core)
```

Comment:
How about for the CM4 core?

Reply:
it is not working now for linkserver, let me forward you a mail